### ISSUE TITLE
SimpleObjectPooling methods

### ISSUE THREAD

**Issue by ntl92bk** [QuocHieuNguyen/ExtenListView#2]
The Get method and the ReturnToPool method should be private. I don't know why do you have to expose it. [SimpleObjectPooling](https://github.com/QuocHieuNguyen/ExtenListView/blob/develop/Assets/ExtenListView/Runtime/ItemLoader/SimpleObjectPooling.cs#L25)

**Comment by QuocHieuNguyen** [comment]
> The Get method and the ReturnToPool method should be private. I don't know why do you have to expose it. [SimpleObjectPooling](https://github.com/QuocHieuNguyen/ExtenListView/blob/develop/Assets/ExtenListView/Runtime/ItemLoader/SimpleObjectPooling.cs#L25)

That's true, the purpose of those methods is wrapped by SpawnItem and DeSpawnItem method, I will update in the next version.
